hello-helm/k8s/templates/configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
As we're not introducing ConfigMaps until the next chapter, we'd better leave this out for now (or duplicate it and remove the config and secret from one version). In your chapter section on 'Update a value in the Helm chart and update', can you make a different change instead? Maybe change the labels or something?